### ISSUE TITLE
install has three dependency conflict on clean create-react-app

### ISSUE THREAD

**Issue by jnorris441** [chakra-ui/chakra-ui#5439]
### Description

expected chakra to install in a clean app

### Link to Reproduction

-

### Steps to reproduce

If this is grotesquely incorrect then please close this

1. use create-react-app@5.0.0 in a new folder which uses React 17.0.2
2. run the command in the readme `npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^5`

### Chakra UI Version

1.7.4

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

```
jasonnorris@Jasons-Mac-mini my-app % npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^5
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: my-app@0.1.0
npm ERR! Found: three@0.136.0
npm ERR! node_modules/three
npm ERR!   peer three@">=0.133" from @react-three/fiber@7.0.25
npm ERR!   node_modules/@react-three/fiber
npm ERR!     peerOptional @react-three/fiber@"*" from framer-motion@5.6.0
npm ERR!     node_modules/framer-motion
npm ERR!       framer-motion@"^5" from the root project
npm ERR!       1 more (@chakra-ui/react)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional three@"^0.135.0" from framer-motion@5.6.0
npm ERR! node_modules/framer-motion
npm ERR!   framer-motion@"^5" from the root project
npm ERR!   peer framer-motion@"3.x || 4.x || 5.x" from @chakra-ui/react@1.7.4
npm ERR!   node_modules/@chakra-ui/react
npm ERR!     @chakra-ui/react@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/jasonnorris/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jasonnorris/.npm/_logs/2022-01-23T23_03_37_551Z-debug.log
```

**Comment by dung-phan** [comment]
I also ran into the same into with a clean up setup. Any solution so far?

**Comment by TimKolberger** [comment]
Fixed with #5499 and release of @chakra-ui/react@1.8.3.
